### PR TITLE
Update nuget version for ManagedServices RP

### PR DESCRIPTION
**Pull request by rajivnandivada**

### CODE REVIEW

**Review by erich-wang — CHANGES_REQUESTED** [review]
Please remove package-lock.json, please also clean up commit history

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (weshaggard) [inline package-lock.json]: Please do not commit this file to the repo.

**Comment by rajivnandivada** [comment]
I found a bug in swagger spec where API version is being treated as a parameter. I will abandon this PR and created another one. Thank you.

**Pull request closed without merging**